src/hypertable.h

```diff
@@ -26,9 +26,20 @@ typedef struct SubspaceStore SubspaceStore;
 typedef struct Chunk Chunk;
 typedef struct Hypercube Hypercube;
 
-#define TS_HYPERTABLE_HAS_COMPRESSION(ht)                                                          \
-	((ht)->fd.compressed_hypertable_id != INVALID_HYPERTABLE_ID)
+enum
+{
+	HypertableCompressionOff = 0,
+	HypertableInternalCompressionTable = 1,
+	HypertableCompressionEnabled = 2,
```

Comment:
Again, I think it makes more sense to make the enabled state `1` and the internal table `2`.

Reply:
Also couldnt hurt to have a text block describing what value this will be on different hypertables in singlenode/multinode scenario

Reply:
Also agree with Erik on the ordering

Reply:
See my earlier comments on why I chose to retain the original mapped values for "compressed" in the "compression_state". Given that enum values should be used  by the C code, the internal values that it gets mapped to should not really matter.  
Will add comments.

Reply:
I agree that it doesn't technically matter for the C code. But the catalog will still show these values and even though the tables there are "internal" it is not uncommon to look into them or do views on them. If we have the opportunity to chose a more logical numbering here, I think it is worth it.